cvat/apps/engine/signals.py

```diff
@@ -17,3 +28,12 @@ def update_task_status(instance, **kwargs):
         db_task.status = status
         db_task.save()
 
+
+@receiver(post_delete, sender=Task, dispatch_uid="delete_task_files_on_delete_task")
+def delete_task_files_on_delete_task(instance, **kwargs):
+    shutil.rmtree(instance.get_task_dirname, ignore_errors=True)
```

Comment:
Looks wrong: instance.get_task_dirname -> instance.get_task_dirname()

Could you please add a test that the bug is fixed?

Reply:
> Looks wrong: instance.get_task_dirname -> instance.get_task_dirname() 

Thanks, I'll fix it. By the way all such functions look like properties.

> Could you please add a test that the bug is fixed?
Could you please give an example?

Reply:
https://github.com/openvinotoolkit/cvat/tree/develop/cvat/apps/engine/tests

Reply:
Properties don't have a verb as a prefix. Usually `task_dirname` is a property and `get_task_dirname` is a function.